utils/release/operator/make_chart.sh

```
@@ -33,6 +33,15 @@ dns_operator_image="quay.io/kuadrant/dns-operator:$dns_operator_version"
 V=$dns_operator_image \
 yq eval '(select(.kind == "Deployment").spec.template.spec.containers[].env[] | select(.name == "RELATED_IMAGE_DNS_OPERATOR").value) = strenv(V)' --inplace $env/config/manager/manager.yaml
 
+# Set desired mcp-gateway images
+mcp_gateway_version=$(mod_version $(yq '.dependencies.mcp-gateway' $env/release.yaml))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)